backend/config/dev.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  "from" : "[FROM EMAIL ID HERE]",
+  "smtpOptions" : {
+  "service": "Gmail",
+  "auth": {
```

Comment:
Put all config parameters in the config file which we already have.

Reply:
What I can see is that there's a config folder. It does contain smaller files to make the code more readable. I've made the file dev.js so that exporting of it happens only when needed, and not at times when I need to access my Database.